model/convert/service_event.go

```diff
@@ -78,6 +79,8 @@ func convertServiceEventEpochSetup(event flow.Event) (*flow.ServiceEvent, error)
 	var dkgPhase3FinalView cadence.UInt64
 	var cdcClusters cadence.Array
 	var cdcParticipants cadence.Array
+	var targetEndTimeUnix cadence.UInt64
```

Comment:
Let's include the unit here [seconds] 
```suggestion
	var targetEndTimeUnix cadence.UInt64 // Unix time [seconds]
```